tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -710,7 +720,7 @@ def after_connect(self):
 
 
 class WdspecProtocol(Protocol):
-    server_cls = None
+    server_cls = None  # type: ClassVar[Optional[Type[WebDriverServer]]]
```

Comment:
`Optional` is needed here because of this code added a few days ago:
https://github.com/web-platform-tests/wpt/blob/c21b0b8ca7b8ba79b185547247f3ccf20081b414/tools/wptrunner/wptrunner/executors/executormarionette.py#L1100-L1101

I didn't add a comment referring to `GeckoDriverProtocol` because it seems guaranteed to go stale, but that's the reason if someone comes looking at this PR.

Reply:
Yeah, this is a design problem. https://github.com/web-platform-tests/wpt/pull/29089 will address that once it's ready.